src/components/NegotiationPosts.vue

```diff
@@ -142,12 +145,15 @@ export default {
       }
     },
     async addMessage() {
+      console.log(this.scope.toUpperCase())
```

Comment:
same as before

Reply:
fixed in 338765dbdacd1bacd4779ccc2b2d75595643e489